tests/test_runner/test_debug_sql.py

```diff
@@ -90,6 +99,7 @@ def test_output_verbose(self):
         ('''SELECT COUNT(*) AS "__count" '''
             '''FROM "test_runner_person" WHERE '''
             '''"test_runner_person"."first_name" = 'subtest-fail';'''),
+        'ERROR: setUpClass (test_runner.test_debug_sql.TestDebugSQL.ErrorClassTest)',
```

Comment:
This patch fixes the issue for me, however new tests work even without it, so I don't think we need them.

Unfortunately, it's more complicated then raising and error in the `setUpTestData()`.

Reply:
OK I figure out why it doesn't fail. `ErrorClassTest` must be the first one, because there is a single instance of `DebugSQLTextTestResult` so `debug_sql_stream` already exists  if we run any test before it. Edits is progress ...

Reply:
Crash depends on the order of added tests, so I moved it to a separate one.

Reply:
Ah yes, of course! Thanks for the new test.